File: Data Structures & Algorithms/max-consecutive-ones/submission-1.rs
impl Solution {
    pub fn find_max_consecutive_ones(nums: Vec<i32>) -> i32 {
        let mut count: i32 = 0;
        let mut ans: i32 = 0;

        for num in nums.iter() {
            if *num != 1 {
                count = 0;
                continue;
            }
            count += 1;
            ans = max(count, ans);
        }

        ans
    }
}

File: Data Structures & Algorithms/is-anagram/submission-1.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        count = {}

        for ss in s:
            if ss in count:
                count[ss] = count[ss] + 1
            else:
                count[ss] = 1
        
        for ts in t:
            if ts not in count:
                return False
            
            if count[ts] == 0:
                return False

            count[ts] = count[ts] - 1

            if count[ts] == 0:
                del count[ts]
        
        return True if len(count) == 0 else False